Gemfile.lock

```diff
@@ -0,0 +1,59 @@
+PATH
```

Comment:
usually libs don't need to check in the Gemfile.lock, any specific reason to do so here?

Reply:
Ah, sorry, no specific reason... Removed it in 942a1c4